pwndbg/gdblib/regs.py

```diff
@@ -174,6 +192,20 @@ def changed(self) -> List[str]:
                 delta.append(reg)
         return delta
 
+    @property
+    @pwndbg.lib.cache.cache_until("stop")
+    def idt(self) -> int:
+        if pwndbg.gdblib.qemu.is_qemu_kernel() and pwndbg.gdblib.arch.current == "x86-64":
```

Comment:
Don't we have `OnlyWhenQemuKernel` and `OnlyWithArch` annotations that can be used instead of if-statements?

Reply:
Works for me. I'm not a huge fan of the duplication of the annotations, but we can always attempt to merge them in the future (definitely don't try to deduplicate in this PR if adds any complexity, I don't want to delay merging it for that).